front-end/.gitignore

```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 .env
+./package-lock.json
```

Comment:
Please don't add package-lock json into .gitignore. Whole purpose of package-lock.json is to guarantee that versions of packages are the same for all developers.